packages/api/src/UserPreferences/UserPreferences.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { Field, ObjectType, InputType } from '@nestjs/graphql';
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn, Unique } from 'typeorm';
+import { IsInt, Min, Max } from 'class-validator';
+
+@ObjectType()
+@Entity()
+@Unique(['userId'])
+export class UserPreferences extends BaseEntity {
+    @PrimaryGeneratedColumn('uuid')
+    @Field()
+    id: string;
+
+    @Column()
+    @Field()
+    userId: string;
+
+    @Column()
+    @Field()
+    @IsInt()
+    @Min(1)
+    @Max(4)
+    practiceGoal: number;
```

Comment:
I do not understand why, but it seems still allows setting number other than 1 - 4 to database.

Reply:
Hmm, interesting. This looks correct according to the offical docs. How are you testing it?

Reply:
@quimgv we should add a test for this

Reply:
On localhost playground, I did mutation:
`mutation {
  updatePreferences(input: {  
    practiceGoal: 7
    why: "secret 4"
    codingAbility: 20
})
  {
    id
    userId
    practiceGoal
    why
    codingAbility
  }
}`

I get successful result, and check the database, the value is set exactly as what is in the request, 7 and 20. 
`{
  "data": {
    "updatePreferences": {
      "id": "8fad1616-d9a2-4681-945c-6c2a21f11065",
      "userId": "421dd1a7-d5d1-4a36-9ef3-14e4a91cb920",
      "practiceGoal": 7,
      "why": "secret 4",
      "codingAbility": 20
    }
  }
}`

Reply:
OK great call. Thanks @Ellie-2020. We will definitely need a test for this then. Not sure why that's happening. Have a further dig, and if you guys can't find it I'll have a look later on

Reply:
I just pull the latest version of the branch and tested. Now it works. I did 3 test using mutation updateUserPreferences:
1. all the 3 parameters with valid value, it execute successful.
2. change practiceGoal value as 8 which is invalid. It failed as expected.

Reply:
3. change  codingAbility value as 90 which is invalid. it failed as expected. 

So it is solved.